f3_test.go

```diff
@@ -37,21 +37,16 @@ var ManifestSenderTimeout = 30 * time.Second
 
 func TestF3Simple(t *testing.T) {
 	t.Parallel()
-	env := newTestEnvironment(t, 2, false)
-
-	env.connectAll()
-	env.start()
+	env := newTestEnvironment(t).withNodes(2).start()
 	env.waitForInstanceNumber(5, 10*time.Second, false)
 }
 
 func TestF3WithLookback(t *testing.T) {
 	t.Parallel()
-	env := newTestEnvironment(t, 2, true)
-	env.manifest.EC.HeadLookback = 20
-	env.updateManifest()
+	env := newTestEnvironment(t).withNodes(2).withManifest(func(m *manifest.Manifest) {
+		m.EC.HeadLookback = 20
```

Comment:
It's now possible to mutate the manifest before constructing the f3 nodes, avoiding the need for a dynamic manifest.